OpenRobertaServer/staticResources/blockly/msg/js/ar.js

```diff
@@ -629,7 +632,6 @@ Blockly.Msg.MENU_PROPERTIES = "properties";  // untranslated
 Blockly.Msg.MENU_RIGHT_CODE_TOOLTIP = "Open/close the source code view.";  // untranslated
 Blockly.Msg.MENU_RIGHT_HELP_TOOLTIP = "Open/close the help view.";  // untranslated
 Blockly.Msg.MENU_RIGHT_INFO_TOOLTIP = "Open/close the program documentation view.";  // untranslated
-Blockly.Msg.MENU_RIGHT_LEGAL_TOOLTIP = "Open/close the legal information view.";  // untranslated
```

Comment:
probably needs another rebase, if those are removed in the PR (or regenerating those files for all langauges)